### PR TITLE
Browser SDK: Add option to transmit browser properties and events to the server

### PR DESCRIPTION
**Pull request by joysonjoy1811**

### CODE REVIEW

File: src/browser/index.html
Comment (vasani-arpit) [inline src/browser/index.html:1]: No need to add this file in codebase. Keep testing environment separate.

File: src/browser/src/core/http.ts
Comment (vasani-arpit) [inline src/browser/src/core/http.ts:1]: I don't think this file will be needed. `browser.ts` should cover this using axios or fetch.

File: src/browser/index.html
Comment (joysonjoy1811) [reply src/browser/index.html:1]: Added the file to .gitignore

File: src/browser/src/core/http.ts
Comment (joysonjoy1811) [reply src/browser/src/core/http.ts:1]: Removed http module from the source code

**Pull request merged**